errors/example_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package xerrors_test
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/m3db/m3x/errors"
+)
+
+func ExampleMultiError() {
+	mutliErr := xerrors.NewMultiError()
+
+	for i := 0; i < 5; i++ {
+		_, err := http.Get("http://example.com/")
+		if err != nil {
+			// Add returns a new MultiError.
+			mutliErr = mutliErr.Add(err)
+		}
+	}
+
+	if err := mutliErr.FinalError(); err != nil {
```

Comment:
same as above: confirm this is actually run.

Reply:
Following up on the previous comment on `// Output:`, I think including it here makes sense. Will update.

Reply:
Updated this, GIthub just didn't notice because this particular line didn't change.

Reply:
nit: Have the `Output` be driven off an error message specified in one of the `err`s